packages/api-explorer/src/form-components/ObjectField.jsx

```diff
@@ -55,13 +55,6 @@ class CustomObjectField extends ObjectField {
             formContext={formContext}
           />
         )}
-        {(uiSchema['ui:description'] || schema.description) && (
```

Comment:
This change came from the original directory I had locally I think we can keep it as we discussed until we figure out how to design readme to be recursive?

Reply:
Ahhh yeah i think we should revert this change until we've fully figured out what the repercussions are and fixed that issue separarely?